applications/GettingStartedWith/HtmlEditor/index.html

```diff
@@ -0,0 +1,53 @@
+<div id="html-editor">
+    <h2>
+        <img src="/Content/images/doc/21_1/htmlEditor/HtmlEditor.png" alt="HtmlEditor">
+        Formatted Text Editor (HTML Editor)
+    </h2>
+    <br>
+    <p>DevExtreme JavaScript HTML Editor is a client-side WYSIWYG text editor that allows its users to format textual and visual content and store it as HTML or Markdown.</p>
+    <p>Supported features:</p>
+    <ul>
+        <li>Inline formats:
+            <ul>
+                <li><strong>Bold</strong>, <em>italic</em>, <s>strikethrough</s> text formatting</li>
+                <li>Font, size, color changes (HTML only)</li>
```

Comment:
It reads weird like "font" and "size" are different characteristics, although "size" is "font size". So let's change the terms a little bit: 

```suggestion
                <li>Typeface, font size, text color changes (HTML only)</li>
```